pages/linux/pacman-key.md

```diff
@@ -0,0 +1,36 @@
+# pacman-key
+
+> Wrapper script for GnuPG used to manage pacman’s keyring.
+> More information: <https://man.archlinux.org/man/pacman-key>.
+
+- Initialize the pacman keyring:
+
+`sudo pacman-key --init`
+
+- Add the default ArchLinux keys:
+
+`sudo pacman-key --populate {{archlinux}}`
+
+- List keys from the public keyring:
+
+`pacman-key --list-keys`
+
+- Add the specified keys:
+
+`sudo pacman-key --add {{path/to/keyfile}}`
+
+- Receive a key from a key server:
+
+`sudo pacman-key --recv-keys "{{key_id}}"`
+
+- Print the fingerprint of a specific key:
+
+`pacman-key --finger "{{key_id}}"`
```

Comment:
Not sure I understand the point of this point. If you're specifying the key id, isn't that the fingerprint already?

Reply:
No, based on the `pacman-key --list-keys`, the `uid` is `first_name last_name <me@example.com>`.
So, You can use `pacman-key --finger me@example.com` or `pacman-key --finger 'first_name last_name'` to see the fingerprint.

Reply:
Ah I see. Thanks for the explanation! I wonder if we could rephrase the `key_id` at all?